src/sentry/deletions/base.py

```diff
@@ -131,7 +132,10 @@ def chunk(self):
         query_limit = self.query_limit
         remaining = self.chunk_size
         while remaining > 0:
-            queryset = list(self.model.objects.filter(**self.query)[:query_limit])
+            queryset = self.model.objects.filter(**self.query)
+            if self.order_by:
+                queryset = queryset.order_by(self.order_by)
+            queryset = list(queryset[:query_limit])
```

Comment:
I implemented the `order_by` thing, but didn't even up using it.

I think the `EventTag` ordering doesn't matter since we are using the deletion code child relation stuff to delete them by ID: 

https://github.com/getsentry/sentry/blob/8330a0938a896d813a342be1e1ca2237da4a6074/src/sentry/deletions/defaults/event.py#L29-L31